Drop-In/src/test/java/com/braintreepayments/api/dropin/AddCardActivityUnitTest.java

```diff
@@ -258,15 +259,15 @@ public void editingAUnionPayCardNumberIsPossible() {
                 .clientToken(base64EncodedClientTokenFromFixture(Fixtures.CLIENT_TOKEN)));
         setup(httpClient);
 
-        setText(mAddCardView, R.id.bt_card_form_card_number, UNIONPAY_CREDIT);
+        setText(mAddCardView, R.id.bt_card_form_card_number, UNIONPAY_19);
```

Comment:
Updated to use a 19 digit Union Pay test card so that the auto-transition occurs for this test.